app/src/main/java/pm/gnosis/heimdall/ui/settings/general/GeneralSettingsActivity.kt

```diff
@@ -72,7 +72,7 @@ class GeneralSettingsActivity : ViewModelActivity<GeneralSettingsContract>() {
 
         disposables += layout_general_settings_feedback_background.clicks()
             .subscribeBy(onNext = {
-                createFeedback()
```

Comment:
can we remove `createFeedback`?

Reply:
we could. I am not sure whether we still need to add a list of installed safes to the email body. If this is not needed anymore, then we can delete it.

Reply:
it is an unused method, right? so I would remove it